tsl/test/expected/compression_bgw.out

```diff
@@ -223,3 +223,158 @@ SELECT COUNT(*) AS dropped_chunks_count
 SELECT add_compression_policy AS job_id
   FROM add_compression_policy('conditions', INTERVAL '1 day') \gset
 CALL run_job(:job_id);
+\i include/recompress_basic.sql
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+CREATE OR REPLACE VIEW compressed_chunk_info_view AS
+SELECT
+   h.schema_name AS hypertable_schema,
+   h.table_name AS hypertable_name,
+   c.schema_name as chunk_schema,
+   c.table_name as chunk_name,
+   c.status as chunk_status, 
```

Comment:
It would be helpful to convert the status to a human readable format with, e.g., a CASE statement. For instance, "compressed", "not_compressed", etc.